RPIPs/RPIP-4.md

```diff
@@ -92,7 +96,7 @@ Snapshot.org is a web3-native platform for token voting and provides several pre
 
 ### RPL Stake and Weight
 
-RPL is required to be effectively staked for voting to prevent vote-buying by participants unaffiliated with the protocol's operations.
+The RPL eligibility requirements help prevent vote-buying by participants unaffiliated with the protocol’s operations.
```

Comment:
Don't we use straight quotes generally?

Reply:
No idea :P

I don't think there's a significant difference and this is the way it was voted in